src/components/views/Basket/Basket.js

```diff
@@ -161,96 +164,21 @@ const Basket = ({
     );
   };
 
-  const renderAddToAgreementButton = () => {
-    const disabled =
-      selectedAgreementId === undefined ||
-      Object.values(selectedItems).find((v) => v) === undefined; // None of the `selectedItems` value's are `true`;
-
-    return (
-      <div>
-        <Layout className="marginTop1">
-          <Button
-            buttonStyle="primary"
-            data-test-basket-add-to-agreement
-            disabled={disabled}
-            onClick={async () => {
-              await handleAddToExistingAgreement(getSelectedItems());
-            }}
-          >
-            <FormattedMessage id="ui-agreements.basket.addToSelectedAgreement" />
-          </Button>
-        </Layout>
-      </div>
-    );
-  };
-
-  const renderAddToAgreementSection = () => {
-    if (!openAgreementsState.length) return null;
+  const renderAddToSelectedAgreementButton = () => {
+    const disabled = Object.values(selectedItems).find((v) => v) === undefined;
```

Comment:
Again, this takes out consideration for the `selectedAgreementId` stat, but we haven't actually gotten rid of that logic.

Either we're changing the pattern to no longer need selectedAgreementId (adn it should be removed entirely) or we're using that pattern (and so shouldn't be removing parts of functionality)